molecule/release-static/molecule.yml

```diff
@@ -38,8 +38,8 @@ platforms:
     image: pulp/molecule_debian10
     command: /sbin/init
   - <<: *platform_base
-    name: fedora-31
-    image: pulp/molecule_fedora31
+    name: fedora-33
+    image: pulp/molecule_fedora33
```

Comment:
I see you changed the distro for all the non-upgrade tests. But what about the upgrade tests with their special manually created images? They are part of the original ticket, although you may work on them in a sub-task if you'd like.
https://pulp.plan.io/projects/pulp/activity?from=2020-11-19

Reply:
new upgrade test image added and used.